bin/commands/bootstrap.js

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+// vim: set ft=javascript:
+
+const chalk = require('chalk');
+const readline = require('readline');
+const path = require('path');
+
+const bootstrap = require('../../lib/bootstrap');
+
+exports.command = 'bootstrap';
+
+exports.desc =
+  'Takes a snapshot of existing space and automatically generate migration scripts';
+
+exports.builder = (yargs) => {
+  yargs
+    .option('access-token', {
+      alias: 't',
+      describe:
+        'CMA token, defaults to your environment variable CONTENTFUL_MANAGEMENT_ACCESS_TOKEN if empty',
+      demandOption: !process.env.CONTENTFUL_MANAGEMENT_ACCESS_TOKEN
+    })
+    .option('space-id', {
+      alias: 's',
+      describe: 'space id to use',
+      demandOption: true
+    })
+    .option('danger-will-robinson-danger', {
+      describe: 'delete all current migrations and create already-applied states for all content types in space'
+    });
+};
+
+exports.handler = (argv) => {
```

Comment:
small refactoring, might make sense to put the deconstruct in the argument
`exports.handler = ({ spaceId, dangerWillRobinsonDanger}) => {`
same thing for the other scripts.